### ISSUE TITLE
Data imbalance

### ISSUE THREAD

**Issue by jiangdizhao** [ultralytics/yolov5#4160]
## ❔Question
Hi,  We have encountered a problem: some classes of our training data are very hard to acquire. It seems that your default hyper parameter "fl_gamma", which is used in focal loss , is set to 0.0. Since focal loss can alleviate data imbalance problem, have you ever tried focal loss during training ? if not, why not?

Best Regard
Dizhao Jiang

**Comment by glenn-jocher** [comment]
@jiangdizhao fl_gamma did not improve COCO results but results will vary by dataset naturally so it is up to you to experiment with the best hyperparameters for your custom task.
